### PR TITLE
Add qiskit-ignis

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/qiskit-ignis/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy >=1.13
+    - python >=3.6
+    - qiskit-terra >=0.15.1
+    - retworkx >=0.8.0
+    - scipy >=0.19,!=0.19.1
+    - setuptools >=40.1.0
```
Comment (carterbox) [inline recipes/qiskit-ignis/meta.yaml:27]: Do you want a run constrained section?

https://github.com/Qiskit/qiskit-ignis/blob/0fb96d52060a488dbad405bb5aca49a2cda1dd1e/setup.py#L80

File: recipes/qiskit-ignis/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+test:
+  imports:
+    - docs
```
Comment (carterbox) [inline recipes/qiskit-ignis/meta.yaml:31]: Docs should not be installed.

**Review by carterbox — CHANGES_REQUESTED** [review]

File: recipes/qiskit-ignis/meta.yaml (hunk 2 above)
Comment (BastianZim) [reply recipes/qiskit-ignis/meta.yaml:31]: Ahh shoot thanks. I'll upstream that.

Comment (BastianZim) [reply recipes/qiskit-ignis/meta.yaml:37]: CI is correctly failing now: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=502148&view=logs&jobId=6f142865-96c3-535c-b7ea-873d86b887bd&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823

**Comment by BastianZim** [comment]
Thanks!

**Pull request merged**